aces/Loris
# [issue_tracker] Resolves special characters in titles

## Brief summary of changes
Resolves how special characters were appearing encoded in the title in the filterable table and the title of the issue itself e.g. `&amp;, &quot;, etc.` instead of `&, ", etc.`

#### Testing instructions (if applicable)

1. Go to Tools -> Issue Tracker
2. Create a new issue and put the special characters `&, <, >, "` in the title; fill out necessary fields -> click Submit 
a. In the main page, the title of the issue in the table should have the characters above as is, and not "`&amp;`", etc. 
3. Click to Edit the issue you just created
a. In the edit form, the title of the issue in the table should again have the characters above, and not "`&amp;`", etc. 

#### Note

This is a CCNA override - https://github.com/aces/CCNA/pull/4032, https://github.com/aces/CCNA/pull/4084
